ExchangeUnion/xud-docker
# chore: update simnet images

This commit will update simnet environment to [latest xud-simnet master](https://github.com/ExchangeUnion/xud-simnet/tree/5d6835a189c42f2c9a95cec366fbcd55b3abc050)

**How to test?**

Run xud script with `-b chore/update-simnet` then select simnet you should have latest xud-simnet experience (auto channel openning is included in this PR).

Closes #326
Reopen previous PR https://github.com/ExchangeUnion/xud-docker/pull/346
